protocol/src/main/java/org/fidoalliance/fdo/protocol/StandardHttpServer.java

```diff
@@ -203,6 +224,35 @@ public void run() {
 
         KeyStoreConfig storeConfig = config.getHttpsKeyStore();
 
+        try {
+          KeyStore ks = loadKeystore();
+          SSLHostConfigCertificate certConfig = null;
+          Certificate cert = ks.getCertificate(storeConfig.getAlias());
+          if (cert != null) {
+            PublicKey publicKey = cert.getPublicKey();
+            if (publicKey instanceof RSAKey) {
+              sslHostConfig = new SSLHostConfig();
+              certConfig = new SSLHostConfigCertificate(
+                  sslHostConfig, SSLHostConfigCertificate.Type.RSA);
+            } else if (publicKey instanceof ECPublicKey) {
+              sslHostConfig = new SSLHostConfig();
+              certConfig = new SSLHostConfigCertificate(
+                  sslHostConfig, Type.EC);
+            }
+          }
+          if (certConfig != null) {
+            certConfig.setCertificateKeystore(ks);
+            certConfig.setCertificateKeyAlias(storeConfig.getAlias());
+            certConfig.setCertificateKeyPassword(storeConfig.getPassword());
+            sslHostConfig.addCertificate(certConfig);
+            httpsConnector.addSslHostConfig(sslHostConfig);
+          }
+        } catch (Exception e) {
+
+          logger.error(e.getMessage());
+        }
+
+        /*
```

Comment:
is this intentional? if so then can you add a comment to indicate why it is commented out?

Reply:
I will remove in next commit